app-backend/api/src/main/scala/scene/Routes.scala

```diff
@@ -396,4 +403,34 @@ trait SceneRoutes
       }
     }
   }
+
+  def getSentinelMetadata(sceneId: UUID, metadataUrl: String): Route =
```

Comment:
It's a bit of a weird api to require a url in the url -- the only maybe better idea I have is to make a query parameter, but I'm not sure how critical friendly API design is here

Reply:
Dear future readers -- not important